pandas/tests/io/formats/test_style.py

```diff
@@ -103,6 +103,12 @@ def test_render(self):
         s.render()
         # it worked?
 
+    def test_render_empty_df(self):
+        empty_df = DataFrame()
```

Comment:
Could you also add a couple checks with

- an index but on columns
- columns but no index

It can be as simple as `DataFrame(columns=['a']).style.render()` and `DataFrame(index=['a']).style.render()`